build-scripts/build_tests.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import pathlib
+import sys
+from typing import TypeVar, Generator, Set, Dict
+import multiprocessing
+
+import ssg.constants
+import ssg.environment
+import ssg.jinja
+import ssg.utils
+import ssg.yaml
+import ssg.templates
+
+SSG_ROOT = str(pathlib.Path(__file__).resolve().parent.parent.absolute())
+JOB_COUNT = multiprocessing.cpu_count()
+T = TypeVar("T")
+
```

Comment:
Now it's really fast! It takes me less than 3 seconds on my laptop to build RHEL 9 tests. Great improvement!